Document what users are supposed to do when text licenses are unavailable. (#115)
microsoft/vcpkg-docs: vcpkg/contributing/maintainer-guide.md

### Install copyright file

> Each port has to provide a file named `copyright` in the folder `${CURRENT_PACKAGES_DIR}/share/${PORT}`. If the license content for a port is available within the sources, this file should be created by a call to [`vcpkg_install_copyright()`](../maintainers/functions/vcpkg_install_copyright.md). `vcpkg_install_copyright` also bundles multiple copyright files if necessary.

Reviewer: ```suggestion
Each port has to provide a file named `copyright` in the folder `${CURRENT_PACKAGES_DIR}/share/${PORT}`. If a port's license content is available within its source files, this file should be created by a call to [`vcpkg_install_copyright()`](../maintainers/functions/vcpkg_install_copyright.md). `vcpkg_install_copyright` also bundles multiple copyright files if necessary.

```

Author: I intentionally did not use this construction because I was worried that describing something as "the port's license" implied the license that applied to the port itself, that is, the directory where `vcpkg.json` and `portfile.cmake` live. We really want some way to say 'the license associated with the content that the port installs'. I think 'the content that the port installs' has been associated with the word 'package' but merging artifacts really made that even more confusing...

What are your thoughts?

Reviewer: I think it's reasonable to change the word "port" to "package" in this context. The package is what ultimately ends up installed on the machine so that way there should be less confusion.

```cmake
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
```

> You might also see old-style manual creation of this file using CMake's built in `file` command. This form is discouraged in favor of `vcpkg_install_copyright` in new ports, but is still acceptable.

Reviewer: ```suggestion
An older method to manually create this file is with CMake's built in `file` command. This is discouraged in favor of `vcpkg_install_copyright` in new ports but is still allowed.

```

```cmake
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

> If the license content is not available in the upstream sources in textual form, such as a license available in PDF only form, `copyright` should contain an explanation as to how a user can find the license requirements, and, if possible a link to the original sources indicating this so that a user can check that it is up to date.

Reviewer: ```suggestion
If the license content in the upstream source files is not in text form (e.g. a PDF file), `copyright` should contain an explanation as to how a user can find the license requirements. If possible, it should also include a link to the original source files indicating this, so users can check if it is up to date.

```
